### PR TITLE
Update mac.en.md

### PR DESCRIPTION
**Pull request by cmbarton**
Fixed download links to GRASS for Mac web page

### CODE REVIEW

File: content/download/mac.en.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ or install the latest available version from <a href="https://ports.macports.org
 <ul>
 <li>
  {{< donateDialog isToggle=true >}}  
-    <a href="https://cmbarton.github.io/grass-mac/_pages/download-grass/"><i class="fa fa-download"></i> Download </a>
+    <a href="[https://cmbarton.github.io/grass-mac/download/](https://cmbarton.github.io/grass-mac/download/)"><i class="fa fa-download"></i> Download </a>
```
Comment (veroandreo) [inline content/download/mac.en.md:36]:
```suggestion
    <a href="https://cmbarton.github.io/grass-mac/download/"><i class="fa fa-download"></i> Download </a>
```
The fix was mixing html and md links formatting

Comment (cmbarton) [reply content/download/mac.en.md:36]: Good point Vero. I was being careful to paste in the link in the format currently used. Do you think that markdown would be better or more reliable here?  If so, we might want to change the entire line to markdown if we can keep the buttons.

Comment (veroandreo) [reply content/download/mac.en.md:36]: Nope, I'd leave it in html that allows us to use fontawesome icons easily

**Comment by cmbarton** [comment]
Hopefully this will fix the links.

**Pull request merged**